src/routes/(app)/members/[studentId]/+page.server.ts

```diff
@@ -104,18 +105,39 @@ export const load: PageServerLoad = async ({ locals, params }) => {
   }
 };
 
-const updateSchema = memberSchema.pick({
-  firstName: true,
-  lastName: true,
-  nickname: true,
-  foodPreference: true,
-  classProgramme: true,
-  classYear: true,
-});
+const updateSchema = memberSchema
+  .pick({
+    firstName: true,
+    lastName: true,
+    nickname: true,
+    foodPreference: true,
+    classProgramme: true,
+    classYear: true,
+  })
+  .partial();
 
 export type UpdateSchema = typeof updateSchema;
 
 export const actions: Actions = {
+  updateFoodPreference: async ({ params, locals, request }) => {
+    const { prisma } = locals;
+    const form = await superValidate(
+      request,
+      z.object({ foodPreference: z.string() }),
+    );
+    if (!form.valid) return fail(400, { form });
+    const { studentId } = params;
+    await prisma.member.update({
+      where: { studentId },
+      data: {
+        foodPreference: form.data.foodPreference,
+      },
+    });
+    return message(form, {
+      message: "Medlem uppdaterad",
```

Comment:
this is a MUCH bigger issue than just this. We basically have ~8 texts throughout the entire project which are translated (except for user content), I did build the component in this pr with i18n in mind, because it was confined to one component. 

This is code in a separate file, so if we were to translate this text, we should reasonably translate the other texts in the same file, and perhaps the components which calls actions in this page, and the pages using those components, and the components within those pages, etc etc etc. I think it's better to have a bigger translation PR which does this page-by-page or feature-by-feature

Reply:
Alright. I was happy to see that the rest of this PR seemed to all be translated and this seemed to be the only exception. But I think you are right that doing it in a more systematic way is more likely to succeed.